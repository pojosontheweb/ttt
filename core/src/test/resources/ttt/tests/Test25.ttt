<%@ page import="java.util.List" %>
<%!
    List<String[]> arr;
%>
this is arrays