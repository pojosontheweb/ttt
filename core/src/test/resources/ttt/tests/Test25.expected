package ttt.tests;

import java.util.List;

public class Test25 extends com.pojosontheweb.ttt.Template {

	private final List<String[]> arr;

	/**
	 * Creates an instance of this template.
	 *
	 */
	public Test25(List<String[]> arr) {
		this.arr = arr;
	}

	@Override
	public void doRender(com.pojosontheweb.ttt.TttWriter out) throws Exception {
		write(out, "this is arrays" );
	}
}
